flang/test/Driver/func-attr.f90

```diff
@@ -0,0 +1,23 @@
+! Test that -mframe-pointer can accept only specific values.
```

Comment:
You seem to be testing a bit more in this file than this comment suggests (e.g. that `-mframe-pointer` will make the compiler insert the new attributes)

Please, could you:
* update the comment to capture what's actually being tested (the `-mframe-pointer` flag)
* rename the file - it's testing a particular flag rather than any "func"tion "attr"ibute